Error: Cannot find module 'lodash'

Lodash isn't listed in the `package.json` dependencies and is resulting in the following error:

```
Error: Cannot find module 'lodash'
Require stack:
- /home/icholy/src/github.com/my_project/node_modules/casbin/lib/cjs/frontend.js
- /home/icholy/src/github.com/my_project/node_modules/casbin/lib/cjs/index.js
```

Looks like it was removed here: https://github.com/casbin/node-casbin/commit/293a852803d5e83562a36bc35cdf48def0f0088b

--- a/package.json
+++ b/package.json
@@ -27,7 +27,6 @@
     "@semantic-release/npm": "^7.1.3",
     "@semantic-release/release-notes-generator": "^9.0.3",
     "@types/jest": "^26.0.20",
-    "@types/lodash": "^4.14.168",
     "@types/node": "^10.5.3",
     "@types/picomatch": "^2.2.2",
     "@typescript-eslint/eslint-plugin": "^4.17.0",

--- a/src/frontend.ts
+++ b/src/frontend.ts
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { concat } from 'lodash';
 import { Enforcer } from './enforcer';
 import { deepCopy } from './util';
 
@@ -53,7 +52,7 @@ export async function casbinJsGetPermissionForUser(e: Enforcer, user?: string): 
     item.unshift('g');
   });
 
-  obj['p'] = concat(policy, groupPolicy);
+  obj['p'] = [...policy, ...groupPolicy];
 
   return JSON.stringify(obj);
 }

--- a/test/model.test.ts
+++ b/test/model.test.ts
@@ -13,7 +13,6 @@
 // limitations under the License.
 // noinspection JSMismatchedCollectionQueryUpdate
 
-import * as _ from 'lodash';
 import { DefaultRoleManager, Enforcer, newEnforcer, newModel } from '../src';
 import { keyMatch2Func, keyMatch3Func, keyMatchFunc } from '../src/util';
 
@@ -236,8 +235,8 @@ function customFunction(key1: string, key2: string): boolean {
 }
 
 function customFunctionWrapper(...args: any[]): boolean {
-  const name1: string = _.toString(args[0]);
-  const name2: string = _.toString(args[1]);
+  const name1 = String(args[0]);
+  const name2 = String(args[1]);
 
   return customFunction(name1, name2);
 }

